### PR TITLE
CPS correction

### PR DESCRIPTION
**Pull request by Bani2791**
### **2.1 Tidy-up**

https://bani--connexcs-docs.netlify.app/customer/cli/

### **2.2 Added the example**: *How to Invert a regular expression*

This will BLOCK all USA Freephone numbers

^((?!^1(800|833|844|855|866|877|888)).)*$

https://bani--connexcs-docs.netlify.app/customer/cli/#how-to-invert-a-regular-expression

### **2.1 Tidy-up**

https://bani--connexcs-docs.netlify.app/customer/did/

### **2.2 Billing> The link of the "packages" should be linked**

**Done**

https://bani--connexcs-docs.netlify.app/customer/did/#billing

### CODE REVIEW

**Comment by Bani2791** [comment]
## **1. Customer > CLI**

**1.1 The documentation should be in series as it is in Control panel eg, CLI,rewrite CLI .....**

**Done**
https://bani--connexcs-docs.netlify.app/customer/cli/

## **2. Customer > Create Customer**

**arranged customer functions**

https://bani--connexcs-docs.netlify.app/customer/customer/

## **3. Customer > Routing > CLI**

**Refer below screenshot."Enter the CLI/ANIs you want to block in the CLI Restrict box". CLI restrict should link back to Provider card>advanced>CLI restrict**

**done**

https://bani--connexcs-docs.netlify.app/customer/cli/#allow-list-numbers-per-provider

## **4. Setup > Anyedge**

Incorporated **Vale** suggestions

https://bani--connexcs-docs.netlify.app/anyedge/anyedge/

File: docs/anyedge/anyedge.md [hunk 1]
```diff
@@ -51,39 +51,39 @@ When a packet traverses NAT, the UDP packet headers are correctly rewritten by y
...
 
-We can use any of the following indicators to detect if NAT is present in the UAC.
+You can use any of the following indicators to detect if NAT is present in the UAC.
 
 + Search the Contact header  field for occurrences of RFC1918 / RFC6598 addresses.
 + Use the "received" test: "address in Via" to compare against the source IP address used for signaling.
 + Search the top-most VIA for the occurrences of RFC1918 / RFC6598 addresses.
 + Search the Session Description Protocol(SDP) for the occurrences of RFC1918 / RFC6598 addresses.
-+ Test if the source port is different from the "port in Via".
++ Test if the source port is different from the "port in Via."
```
Comment (digipigeon) [inline docs/anyedge/anyedge.md:60]: Are you sure about this correction?

File: docs/anyedge/anyedge.md [hunk 2]
```diff
@@ -108,7 +108,7 @@ If a packet fails to validate, you can choose how to handle it. You can handle t
 + No "To" header.
  No "Via" header.
 + Request URI parse error.
-+ Bad hostname in "R-URI".
++ Bad hostname in "R-URI."
```
Comment (digipigeon) [inline docs/anyedge/anyedge.md:111]: Are you sure about this one?

File: docs/anyedge/anyedge.md [hunk 3]
```diff
@@ -16,32 +16,32 @@ Click the :material-plus: button to set the following:
...
+After AnyEdge configuration is complete, click **:material-plus:** next to Domains to configure a specific domain. Complete it with the same settings as Configure AnyEdge. It also provides an extra Transport Layer Security (TLS) configuration for SIP protection.
 
 ### AnyEdge Destinations
 
 Click :material-plus: button to specify the Destination IP and one or more Limit (Primary) and Backup (Secondary) Zones.
 
 ## Capabilities
 
-The ConnexCS **AnyEdge** load balancer is a high-performance application designed for maximum throughput using multiple cores. Combined with global PoPs and detailed metrics, we’ve got you covered even if you have requirements that exceed 10K calls per second.
+The ConnexCS **AnyEdge** load balancer is a high-performance application. It is designed for utmost throughput using multiple cores. Combined with global PoPs and detailed metrics, we’ve got you covered even if you have requirements that exceed 10K calls per second.
```
Comment (digipigeon) [inline docs/anyedge/anyedge.md:38]: utmost - is this a suitable (accessible) word for the documentation?

File: docs/anyedge/anyedge.md [hunk 4]
```diff
@@ -51,39 +51,39 @@ When a packet traverses NAT, the UDP packet headers are correctly rewritten by y
...
 + Search the Session Description Protocol(SDP) for the occurrences of RFC1918 / RFC6598 addresses.
-+ Test if the source port is different from the "port in Via".
++ Test if the source port is different from the "port in Via."
 + Compare the "address in Contact" with the source IP address used for signaling.
 + Compare the "Port in Contact" with the source port used for signaling.
 
 ### Inbound Proxy / Dispatcher / Load Balancer
 
-The primary use case for **AnyEdge** is to disseminate calls to a pool of SIP Servers. These can be configured using the following call strategies:
+The primary use case for **AnyEdge** is to disseminate calls to a pool of SIP Servers. Configure the servers using the following call strategies:
```
Comment (digipigeon) [inline docs/anyedge/anyedge.md:66]: This does not read well.

**Review by digipigeon — CHANGES_REQUESTED** [review]

File: docs/anyedge/anyedge.md (hunk 2 above)
Comment (Bani2791) [reply docs/anyedge/anyedge.md:111]: Yes Sir, the corrections are done according to the "Vale" suggestions.

File: docs/anyedge/anyedge.md (hunk 4 above)
Comment (Bani2791) [reply docs/anyedge/anyedge.md]: Rewritten as : 
The primary use case for *AnyEdge* is to distribute calls to the SIP Servers. You can use the following algorithms to distribute the calls:

**Review by digipigeon — APPROVED** [review]

**Pull request merged**